src/clusterfuzz/_internal/bot/fuzzers/afl/launcher.py

```diff
@@ -486,12 +503,26 @@ def __init__(self,
     for env_var, value in six.iteritems(config.additional_env_vars):
       environment.set_value(env_var, value)
 
-    self.showmap_output_path = os.path.join(fuzzer_utils.get_temp_dir(),
-                                            self.SHOWMAP_FILENAME)
+    self._showmap_output_path = None
     self.merge_timeout = engine_common.get_merge_timeout(DEFAULT_MERGE_TIMEOUT)
     self.showmap_no_output_logged = False
     self._fuzz_args = []
 
+  @property
+  def showmap_output_path(self):
```

Comment:
I got rid of this in a recent change. Are you sure this needs to be done lazily?

Reply:
Please answer this question